app/views/books/_form.html.erb

```diff
@@ -0,0 +1,29 @@
+<%= form_with model: @book do |form| %>
+  <div>
+    <%= form.label :title, "Title" %>
+    <%= form.text_field :title, required: true, placeholder: "Enter book title" %>
+    <% @book.errors.full_messages_for(:title).each do |message| %>
+      <div><%= message %></div>
```

Comment:
You might want to look into [aria-describedby](https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA21#example-2-identifying-errors-in-data-format) for associating each error with the relevant field